data/tlg0007/tlg068/tlg0007.tlg068.perseus-eng3.xml

```diff
@@ -11,8 +11,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
    <teiHeader xml:lang="eng"><!--status="new"-->
       <fileDesc>
          <titleStmt>
-            <title>How the Young Man Ought to Study Poetry</title>
-            <title type="sub">Machine readable text</title>
+            <title rend="italic">How the Young Man Ought to Study Poetry</title>
```

Comment:
No italic in the header please. (May have been a global change? Be careful about those.)